### ISSUE TITLE
Weston-launch seems to interfere with password change on first boot

### ISSUE THREAD

**Issue by eliasbakken** [intelligent-agent/Refactor#224]
It looke like weston-launch is interfering with the password change routine on Recore. This appears repeatedly in the log, possibly until the password is changed. 
```
weston-launch[1398]: Rejected - it is WAY too short. Try again.
weston-launch[1398]: stty: 'standard input': Inappropriate ioctl for device
```

**Comment by eliasbakken** [comment]
This is related to the fact that weston is running as root on a tty. It should probably be run as a different user under a virtual terminal similar to this: https://gist.github.com/rkitover/78af03c9bd9ec0ab8da493c505bfe1c1

**Comment by eliasbakken** [comment]
Fixed in PR#226
